tests/e2e/yamcs/limits.e2e.spec.js

```diff
@@ -215,3 +303,13 @@ async function assertLimitLinesExistAndAreVisible(page) {
         await expect(page.locator('.c-plot-limit-line').nth(i)).toBeVisible();
     }
 }
+
+/**
+ * Asserts that limits are cleared
+ * @param {import('@playwright/test').Page} page
+ */
+async function assertLimitsAreCleared(page) {
+    const runTimeLimitChangeResponse = await page.request.get('http://localhost:8090/api/mdb-overrides/myproject/realtime/parameters/myproject/Detector_Temp');
```

Comment:
We should replace `http://localhost:8090/` with a variable named "YAMCS_URL" so that these tests can be run against banner.

Reply:
OK